src/plugins/intel_npu/src/common/include/intel_npu/common/sync_infer_request.hpp

```diff
@@ -22,7 +23,9 @@ namespace intel_npu {
  */
 class SyncInferRequest : public ov::IInferRequest {
 public:
-    explicit SyncInferRequest(const std::shared_ptr<const ICompiledModel>& compiledModel, const Config& config);
+    explicit SyncInferRequest(const std::shared_ptr<const ICompiledModel>& compiledModel,
+                              const std::shared_ptr<IGraph>& graph,
```

Comment:
Can we remove the "graph" from here and add a getter in the iCompiledModel?

Reply:
done